src/test/java/io/github/wasabithumb/jtoml/TomlConfigurationLoaderTest.java

```diff
@@ -12,14 +14,16 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-class ConfigurateTest {
+class TomlConfigurationLoaderTest {
 
     @Test
     void simple(final @TempDir Path tempDir) throws IOException {
         final Path configPath = tempDir.resolve("simple.toml");
 
         final TomlConfigurationLoader loader = TomlConfigurationLoader.builder()
                 .path(configPath)
+                .set(JTomlOption.LINE_SEPARATOR, LineSeparator.LF)
```

Comment:
Not sure if configurate does this by default with other loaders, but I will be setting it for resource-factory for reproducible builds